example/conf.py

```diff
@@ -204,3 +204,6 @@ def get_git_revision():
      'Miscellaneous'),
 ]
 
+html_context={
+    "feedback":True,
```

Comment:
If we turn to False, it removes the plugin right?

Reply:
Yes